doc/procedures/jira.rst

```diff
@@ -55,17 +56,14 @@ Sprints
     * Notify Sprint Lead of time frame work is expected to be resolved
 
 * Keep commitment to approximately 8 story points (2 weeks)
-* Monthly Retrospective Meetings
 
-    * Go over accomplishments
-    * Go over any process issues that can be improved
-    * Explain future schedule and path
+    * Each sprint assumes 2 story points for meetings and other administrative tasks
 
 Task Process
 ============
 
 * Do not create Epics.
-  That is the responsiblity of the Sprint Lead.
+  That is the responsibility of the Sprint Lead or manager.
```

Comment:
Actually, only the Manager should do it.  and of course... you capitalize Manager.  He is a very important person.  Well, at least according to him, anyway.